### PR TITLE
Add documentation for Backup integration

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Related:
- <https://github.com/home-assistant/core/pull/66395>
- <https://github.com/home-assistant/frontend/pull/11671>
- <https://github.com/home-assistant/brands/pull/3157>
- <https://github.com/home-assistant/home-assistant.io/pull/21612>

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66395
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/backup.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Backup
+description: System Health allow you to create backups on Core and Container installations.
```
Comment (balloob) [inline source/_integrations/backup.markdown]:
```suggestion
description: Allow creating backups of container and core installations.
```

File: source/_integrations/backup.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ha_category:
+  - Other
+ha_release: 2022.3
+ha_quality_scale: internal
+ha_domain: backup
+---
+
+The Backup integration allow you to create and download backups for your Home Assistant Core and Home Assistant Container installation, this backup file can be used if you migrate to Home Assistant Operating System.
```
Comment (balloob) [inline source/_integrations/backup.markdown]:
```suggestion
The Backup integration allow you to create and download backups for your Home Assistant Core and Home Assistant Container installations. This backup file can be used if you migrate to Home Assistant Operating System.
```

File: source/_integrations/backup.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<div class="note">
+
+If you use Home Assistant Operating System or Home Assistant Supervised, this integration is not for you, you already have [Backup functionaloty with the Supervisor](/common-tasks/os/#backups).
```
Comment (balloob) [inline source/_integrations/backup.markdown]:
```suggestion
If you use Home Assistant Operating System or Home Assistant Supervised, [back up functionality is already built-in]](/common-tasks/os/#backups).
```

File: source/installation/index.html [hunk 4]
```diff
@@ -228,13 +228,15 @@ <h2>Compare Installation Methods</h2>
...
+        <td><a href="/common-tasks/os/#backups" target="_blank">Backups</a></td>
+        <td>✅</td><td>✅<sup>1</sup></td><td>✅<sup>1</sup></td><td>✅</td>
     </tr>
     <tr>
         <td>Managed OS</td>
         <td>✅</td><td>❌</td><td>❌</td><td>❌</td>
     </tr>
 </table>
 
+<sub>1: Backups for Home Assistant Core and Home Assistant Container is provided by the <a href="/integrations/backup" target="_blank"><code>backup</code> integration</a>.</sub>
```
Comment (balloob) [inline source/installation/index.html:240]: Wonder if we can add a My link that works for both hassio + backup integration.

Comment (ludeeus) [reply source/installation/index.html:240]: the frontend handles that, so yes

Comment (ludeeus) [reply source/installation/index.html:240]: This route to the documentation not to the instance so not sure it should be here at all. but for other places, we can do it.

File: source/_integrations/backup.markdown [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+title: Backup
+description: Allow creating backups of container and core installations.
+ha_category:
+  - Other
+ha_release: 2022.3
```
Comment (ludeeus) [inline source/_integrations/backup.markdown]: Will not make 2022.3, version number needs adjustment

**Review by frenck — APPROVED** [review]

**Pull request merged**